test/unit/catalog.js

```diff
@@ -101,20 +101,20 @@ describe("Catalog", function () {
         assert.equal(catalog.getPlural(2, "Bird", "Birds"), "[MISSING]: Birds");
     });
 
-    it("Can return an interpolated string", function () {
+    it("Can not interpolate strings", function () {
         var strings = { "Hello {{name}}!": "Hallo {{name}}!" };
         assert.deepEqual(catalog.strings, {});
         catalog.setCurrentLanguage("nl");
         catalog.setStrings("nl", strings);
-        assert.equal(catalog.getString("Hello {{name}}!", { name: "Andrew" }), "Hallo Andrew!");
+        assert.equal(catalog.getString("Hello {{name}}!", { name: "Andrew" }), "Hallo {{name}}!");
```

Comment:
This is a break in behavior that will certainly break applications. I'm not sure we can even change this.

Reply:
I thought so. But this is also the root of all evil, as it allows the execution of JavaScript code:
```catalog.getString("Hello {{name}}!", { name: "<script>document.write('fa' + 'il');</script>" })```
I didn't find a way to tell $interpolate to be XSS safe...

Reply:
Hmmmm, that's annoying.

I'll need to give this some thought. There's some pretty big applications and I don't want to break things, but obviously security matters.

Reply:
That's legitimate. If anything, I can try to change the commit such that the getString method functionality is not broken, but the XSS e2e test still works. I'm pretty positive that I can do that.